docs/diagrams/transfer-push-consumer.puml

```diff
@@ -113,29 +86,11 @@ return Request in Process Message
 return
 Dispatcher -> TransferStore --: store TransferProcess
 
-' Notification
-...
-Provider -> IDS ++: (Provisioned) Notification Message
-IDS -> TransferManager -- : notify about event on provider (provisioned)
-TransferManager -> TransferStore: store TransferProcess
-
 ' Transfer
 ...
```

Comment:
It is true that there may be no temporal correlation between some state changes. This would depend on the specific transfer state change and maybe even the extensions, that implements a transfer interface, too.

For simplicity I would not separate them in different diagrams or sections. Instead the dotted lines are now removed. I think the complexity of our transfer process is probably something we should not try to address in detail in any diagram, as it would cause too much confusion. 

@alexandrudanciu I hope this is fine, too.